kedro-org/kedro-viz
# Fix Metadata timestamp for Shareable Viz

## Description

Fix for timestamp not appearing in shareable viz

## Development notes

* Moved the shareable url metadata inside kedro pipeline div to make it visible

## QA notes

* Deploy Kedro Viz and you should see the version and deployed timestamp on the screen
<img width="1559" alt="image" src="https://github.com/kedro-org/kedro-viz/assets/87735534/cde3a88b-236f-4255-a510-bba176adc3f6">

## Checklist
